reference/imagetexture.md

```diff
@@ -57,8 +57,11 @@ clamped in the *s* direction to lie within the [0.0,1.0] range. The `repeatT`
 field is analogous to the `repeatS` field.
 
 The `filtering` field defines the level of quality obtained from filtering
-techniques applied to the texture. This parameter ranges from 0 to 5, 0
-corresponding to no filtering, only performing simple nearest-neighbor pixel
+techniques applied to the texture. This parameter ranges from 0 to an integer
+increment relative to the maximum supported anisotropy level, which is usually 5.
```

Comment:
It's not always the maximum value, only in the case where it is related to the max supported level.